.github/workflows/release-dev-ci-cd.yml

```
@@ -0,0 +1,156 @@
+          tags: |
+            ${{ env.IMAGE }}:${{ steps.tags.outputs.dev_tag }}
+            ${{ env.IMAGE }}:${{ steps.tags.outputs.sha_tag }}
+
+  deploy-dev:
+    if: github.event_name == 'push'
+    needs: build-and-push
+    runs-on:
+      - self-hosted
+      - fittoring-be
+    env:
+      IMAGE: fittoring/fittoring   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)